dev/io.openliberty.persistence.3.1.thirdparty/resources/org/eclipse/persistence/internal/localization/i18n/LoggingLocalizationResource.nlsprops

```diff
@@ -169,9 +169,9 @@ elements={0}{1} elements
 
 enable_batch_operations_as_true_not_supported=WLS native CMP setting "enable-batch-operations-as-true" is not directly supported in EclipseLink CMP. See migration doc for details.
 
-encryptor_script_description=This application internally decrypts an old encrypted password that was used by a previous EclipseLink version and encrypts it by the latest algorithm.
+encryptor_script_description=This application internally decrypt old encrypted password used by some previous version EclipseLink and encrypt it by latest algorithm.
```

Comment:
Please remove this change.... this change will undo the changes requested by ID during the last up date from 4.0.2 to 4.0.3.... see comments in : https://github.com/OpenLiberty/open-liberty/pull/28824